components/infobox/commons/infobox_person.lua

```diff
@@ -51,19 +51,33 @@ local _COUNTRIES_EASTERN_NAME_ORDER = {
 	'South Korea',
 	'Cambodia'
 }
-local STATUS_INACTIVE = 'inactive'
-local STATUS_BANNED = 'banned'
-local STATUS_RETIRED = 'retired'
+---@enum PlayerStatus
+local Status = {
+	ACTIVE = 'Active',
+	INACTIVE = 'Inactive',
+	RETIRED = 'Retired',
+	DECEASED= 'Deceased',
+}
+Status['PASSED AWAY'] = Status.DECEASED
```

Comment:
```suggestion
Status.PASSED_AWAY = Status.DECEASED
```
But I question why this is needed?

Reply:
Let's strandardize unless there's any  objections to it (CC: @iMarbot)

Reply:
https://discord.com/channels/93055209017729024/878766918880874536/1187449443604254762

waiting a few days on a result and depending on what we get we standardize to that (imo)

Reply:
result of the vote:
![IMG_4866](https://github.com/Liquipedia/Lua-Modules/assets/75081997/f6e8943e-2ce8-439f-ad7d-c4ca435e958e)

will look into it "later this year" XD